src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -823,6 +843,7 @@ def add_plane_segmentation_to_nwbfile(
         include_roi_acceptance=include_roi_acceptance,
         mask_type=mask_type,
         iterator_options=iterator_options,
+        quality_metrics=quality_metrics,
```

Comment:
We should call this segmentation_extractor_properties because it is not only quality metrics.